### PR TITLE
dm: add update validation description

### PR DESCRIPTION
**Pull request by lichunzhu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add documention for feature `Real-time Checkpoint Update upon Completion of Incremental Data Validation`.

### Which TiDB version(s) do your changes apply to? (Required)
v7.5

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-continuous-data-validation.md [hunk 1]
```diff
@@ -243,6 +247,25 @@ Flags:
 
 用法可参考 [`dmctl validation start` 命令](#方法-2通过-dmctl-开启)。
 
+
+## 设置增量校验切换点
+
+如果你需要希望增量校验在积攒数据同步到某个位点后立即进行校验完整性，以便于后续业务做数据库切换，可以使用 `validation update` 命令：
```
Comment (ran-huang) [inline dm/dm-continuous-data-validation.md:253]: 请确认下这样表述正确吗？

```suggestion
在将后续业务切换到另一个数据库前，有时需要在数据同步到特定位置后立即进行增量数据校验，以确保数据完整性。此时，你可以将这个特定位置设置为增量校验的切换点。

要设置增量校验的切换点，使用 `validation update` 命令：
```

File: dm/dm-continuous-data-validation.md [hunk 2]
```diff
@@ -243,6 +247,25 @@ Flags:
...
+
+Flags:
+      --cutover-binlog-gtid string   specify the cutover binlog gtid for validation, only valid when source config's gtid is enabled, e.g. '1642618e-cf65-11ec-9e3d-0242ac110002:1-30'
+      --cutover-binlog-pos string    specify the cutover binlog name for validation, should include binlog name and pos in brackets, e.g. '(mysql-bin.000001, 5989)'
+  -h, --help                         help for update
+```
+
+* `--cutover-binlog-gtid`：指定希望 validator 立即校验的位置，格式是：1642618e-cf65-11ec-9e3d-0242ac110002:1-30。
+* `--cutover-binlog-pos`：指定希望 validator 立即校验的位置，格式是：(mysql-bin.000001, 5989)。
+* `task-name`：需要立即补充增量数据校验位点的任务名，该项**必须**填写。
```
Comment (ran-huang) [inline dm/dm-continuous-data-validation.md:267]:
```suggestion
* `--cutover-binlog-gtid`：指定希望 validator 立即校验的位置，格式是：`1642618e-cf65-11ec-9e3d-0242ac110002:1-30`。仅在上游开启 GTID 时生效。
* `--cutover-binlog-pos`：指定希望 validator 立即校验的位置，格式是：`(mysql-bin.000001, 5989)`。
* `task-name`：需要立即补充增量数据校验位点的任务名，该项**必须**填写。
```

File: dm/dm-continuous-data-validation.md [hunk 3]
```diff
@@ -243,6 +247,25 @@ Flags:
...
+  dmctl validation update [task-name] [flags]
+
+Flags:
+      --cutover-binlog-gtid string   specify the cutover binlog gtid for validation, only valid when source config's gtid is enabled, e.g. '1642618e-cf65-11ec-9e3d-0242ac110002:1-30'
+      --cutover-binlog-pos string    specify the cutover binlog name for validation, should include binlog name and pos in brackets, e.g. '(mysql-bin.000001, 5989)'
+  -h, --help                         help for update
+```
+
+* `--cutover-binlog-gtid`：指定希望 validator 立即校验的位置，格式是：1642618e-cf65-11ec-9e3d-0242ac110002:1-30。
+* `--cutover-binlog-pos`：指定希望 validator 立即校验的位置，格式是：(mysql-bin.000001, 5989)。
```
Comment (ran-huang) [inline dm/dm-continuous-data-validation.md:266]: 这两个 flag 是可选的吗？如果同时配置了 gtid 和 pos，哪个生效

**Review by D3Hunter — APPROVED** [review]

File: dm/dm-continuous-data-validation.md [hunk 4]
```diff
@@ -243,6 +247,27 @@ Flags:
 
 用法可参考 [`dmctl validation start` 命令](#方法-2通过-dmctl-开启)。
 
+
```
Comment (Oreoxmt) [inline dm/dm-continuous-data-validation.md:250]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**